geps/gep-1709.md

```diff
@@ -266,6 +280,18 @@ VERSION file contains the semver of the latest release and is manually bumped at
 release time. The script hack/verify-all.sh will be updated to ensure the
 generated `.go` file is up to date with the VERSION file.
 
+### Multiple reports merge
+
+Every profile gives the possibility to specify multiple modes and
+implementation versions, but the testing process is related to a specific mode
+and a specific version. In order to produce reports with multiple modes and versions,
+a proper flag `merge-report-with` can be set when running the certification process.
+The flag value is taken, and if no file with such a name exists, it gets created
+and the report is written therein. If the file already exists, the suite merges
+the current report with the given one. By proceeding this way, it is possible to
+iteratively run multiple times the certification suite with different setups
+(modes and versions) and provide a unique report.
+
```

Comment:
I think it's a minor thing, but it might be helpful to add a default behavior of not overwriting data, and enable a flag:

```suggestion

> **Warning**: By default trying to merge a new report that contains a mode that already exists in the 
> existing report specified by `merge-report-with` will result in an error, to avoid automatically deciding
> which report is the one the user actually intended to provide and then deleting data. Implementors
> will need to manually remove a mode from an existing report if they wish to override, or they can use
> the `merge-report-overwrite-existing` flag to explicitly indicate this is what they want.
```

Reply:
Yeah, it makes sense!